Makefile

```diff
@@ -1,30 +1,30 @@
-GOLANGCI_LINT_VERSION = $(shell head -n 1 .golangci.yml | tr -d '\# ')
-TMPDIR ?= /tmp
-
-all: build
-
-build :
-	go build
-
-format :
-	find . -name '*.go' -exec gofmt -s -w {} +
-
-ci-like-lint :
-	@docker run --rm -t -v $(shell pwd):/app \
-		-v $(TMPDIR)/golangci-cache-$(GOLANGCI_LINT_VERSION):/golangci-cache \
-		--env "GOLANGCI_LINT_CACHE=/golangci-cache" \
-		-w /app golangci/golangci-lint:$(GOLANGCI_LINT_VERSION) \
-		make lint
-
-lint :
-	golangci-lint run --out-format=tab --new-from-rev master ./...
-
-tests :
-	go test -race -timeout 210s ./...
-
-check : ci-like-lint tests
-
-container:
-	docker build --rm --pull --no-cache -t grafana/k6 .
-
-.PHONY: build format ci-like-lint lint tests check container
+GOLANGCI_LINT_VERSION = $(shell head -n 1 .golangci.yml | tr -d '\# ')
+TMPDIR ?= /tmp
+
+all: build
+
+build :
+	go build
+
+format :
+	find . -name '*.go' -exec gofmt -s -w {} +
+
+ci-like-lint :
+	@docker run --rm -t -v $(shell pwd):/app \
+		-v $(TMPDIR)/golangci-cache-$(GOLANGCI_LINT_VERSION):/golangci-cache \
+		--env "GOLANGCI_LINT_CACHE=/golangci-cache" \
+		-w /app golangci/golangci-lint:$(GOLANGCI_LINT_VERSION) \
+		make lint
+
+lint :
+	golangci-lint run --out-format=tab --new-from-rev master ./...
+
+tests :
+	go test -race -timeout 210s ./... -vet=off

```

Comment:
Just a small note here that this affects runs for all operation systems which isn't good, but I guess in that case it's fine as long as we have the linter altogether (`make check`)

Reply:
@olegbespalov Do you want me to add some check related to underlying OS?

Reply:
No, I guess in the current case, it will be over. Just share the opinion that in some cases we can do that.